extruct/w3cmicrodata.py

```diff
@@ -119,10 +121,25 @@ def _extract_properties(self, node, items_seen, base_url):
                 yield p, v
 
     def _extract_property_refs(self, node, refid, items_seen, base_url):
-        for prop in node.xpath("id($refid)/descendant-or-self::*[@itemprop]", refid=refid):
-            for p, v in self._extract_property(
-                    prop, items_seen=items_seen, base_url=base_url):
+        ref_node = node.xpath("id($refid)[1]", refid=refid)
```

Comment:
As far as I understand, if there were several elements with the same ``id``, they were all extracted, but now just the first one is extracted? I realise that there supposed to be only one element with each ``id`` so this seems fine.

Reply:
I see... I went to standard: https://www.w3.org/TR/html401/struct/global.html#h-7.5.2

> id = name [CS]
> This attribute assigns a name to an element. This name must **be unique in a document**.

So I think is safe to spect just one.